test/puppetlabs/ring_middleware/core_test.clj

```diff
@@ -278,7 +279,7 @@
          :endpoint      "/hello-proxy"
          :target-endpoint "/hello"}
         (let [response (http-get "http://localhost:10000/hello-proxy/wrong-host")]
-          (is (= (:status response 502))))))
+          (is (= (:status response) 500)))))
```

Comment:
[TK-228](https://tickets.puppetlabs.com/browse/TK-228) has been opened to track this bug